module/settings.mjs

```diff
@@ -28,6 +28,20 @@ export function registerSystemSettings() {
     }
   });
 
+  game.settings.register("dnd5e", "attackRollCheckVisibility", {
+    name: "SETTINGS.5eAttackRollCheckVisibility.Name",
+    hint: "SETTINGS.5eAttackRollCheckVisibility.Hint",
+    scope: "world", // Applies to the entire world, not just individual users
+    config: true, // Shows this setting in the Configure Settings menu
+    default: "gm", // Sets "gm" as the default value
+    type: String, // Specifies that the setting is a string choice
+    choices: {
+      gm: "SETTINGS.5eAttackRollCheckVisibility.GM",
+      players: "SETTINGS.5eAttackRollCheckVisibility.Players",
+      players_hideac: "SETTINGS.5eAttackRollCheckVisibility.PlayersHideAC"
```

Comment:
To match the wording changes mentioned above and keep the order similar to `Challenge Visibility`:

```suggestion
      all: "SETTINGS.5eAttackRollCheckVisibility.All",
      hideAC: "SETTINGS.5eAttackRollCheckVisibility.HideAC",
      none: "SETTINGS.5eAttackRollCheckVisibility.None"
```